src/reversediff.jl

```diff
@@ -2,23 +2,29 @@ module ReverseDiffX
 
 export NotTracked
 
-using MacroTools, LinearAlgebra, ..ReverseDiff, StaticArrays
+using LinearAlgebra
+using ..ReverseDiff
+using StaticArrays
+using ZygoteRules
+using Distributions
+using PDMats
```

Comment:
Are these packages guaranteed to be available? Are they needed for the other packages? Maybe it would be good to move all imports to the Requires blocks for consistency reasons and to spot easier what dependencies are needed and loaded.

Reply:
The ReverseDiff part was already separated from the rest of DAD, but contained its own `ReverseDiffX` package which loads a bunch of dependencies. This part is just reformatted by me (sorry for that), but all these dependencies are guaranteed to be available, yes.

Reply:
Do you think it makes sense to refactor the ReverseDiff part or should we keep it like this for now?

Reply:
Ah I think the reason for this additional module was that all/most of this code should actually go into ReverseDiff and moved upstream. Maybe leave it in a separate module for now (so it's more clearly separated), but put everything in one file? In any case, this is not a critical issue I guess.